### DISCUSSION TITLE
Failed to Import Keys

### DISCUSSION THREAD

**Discussion by DRakox310** [skyline-emu/skyline#333] [Q&A] [answered]
When I try to load the Keys, I get the error "Failed to import the keys". I have an LG G8thinq and have tried Keys from various sources. Does anyone have the same problem?

**Comment by iniMerchant64** [comment, marked as answer]
Sorry but since you publicly admitted to being a pirate by stating that you tried to use keys from "varius sources", we cannot provide you support as a user. Also please reffrain from opening issues without previously asking in the discord server
